chart/templates/rbac.yaml

```diff
@@ -0,0 +1,283 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    {{- include "druid-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: druid-operator
+    app.kubernetes.io/part-of: druid-operator
+  name: {{ include "druid-operator.fullname" . }}-leader-election-role
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: {{ include "druid-operator.fullname" . }}-manager-role
+rules:
+- apiGroups:
+    - ""
+  resources:
+    - configmaps
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - ""
+  resources:
+    - events
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - ""
+  resources:
+    - persistentvolumeclaims
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - ""
+  resources:
+    - pods
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - ""
+  resources:
+    - services
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - apps
+  resources:
+    - deployments
```

Comment:
```suggestion
    - deployments
    - statefulsets
```

Reply:
I can do that, but this YAML are generated bu Kubebuilder, so next time we will generate the YAMLs it will be like that again.

Reply:
kubebuilder now generate the helm chart too ? 👀

Reply:
if so just resolve all the comments on the helm chart, I'll check the doc too :D

Reply:
Oh, I wish hehe. I just mean that he generates RBAC, and what I did was just copy-paste what he generated - that means that if I will do these changes, the next time that someone will change permissions, he will need to manually do these changes again.

Reply:
@cyril-corbon we can talk about it on slack if you want it will be faster :)

Reply:
We might want to check the controller-gen tags on which generate rbac. 
here https://github.com/datainfrahq/druid-operator/blob/master/controllers/druid/druid_controller.go#L41

Reply:
Yes, this is where all the manager's permissions are being generated from. We can removed whatever we want.